core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -170,11 +170,17 @@ private[spark] class BlockManager(
   // service, or just our own Executor's BlockManager.
   private[spark] var shuffleServerId: BlockManagerId = _
 
+  private val registrationTimeout =
+    conf.getTimeAsMs("spark.shuffle.registration.timeout", "5s")
```

Comment:
For new configurations, should we be putting these into the `config` package object? See https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/package.scala and https://github.com/apache/spark/pull/10205